apps/backend/src/modules/projects/project.repository.ts

```diff
@@ -51,10 +61,11 @@ class ProjectRepository implements Repository {
 			query.whereILike("name", `%${name}%`);
 		}
 
-		const projects = await query.execute();
+		const { results, total } = await query.page(page, pageSize);
```

Comment:
Maybe lets add `.execute()` to the end?

Reply:
The query is executed when you call .page(). Why do we need an additional process?